linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#Setup arguments
+
+#Main code
+#check if docker_deamon is running - otherwise start run
+systemctl status docker || systemctl start docker
+
+#check if user wants to create new psql table
+if [ $1 == "create" ]; then
+  #list all docker containers and filter for jrvs-psql and then counts words in name
+  if  [ "$(docker ps -a -f name=jrvs-psql | wc -l)" == "2" ]; then
+    echo "Error: Container already exists"
+    exit 1
+
+  elif [ "$#" != 4 ]; then
+    echo "Error: database username or password not passed"
+    exit 1
+
+  else
+    docker pull postgres
+    docker volume create pgdata
+    export db_user=$2
+    export db_pass=$3
```

Comment:
environment vars are not necessary here. please review my env video posted on slack today

Reply:
fixed